README.md

```diff
@@ -826,3 +826,10 @@ $ oc extract cm/masters-scan-ip-10-0-174-253.ec2.internal-pod
 
 Note that if the results are too big for the ConfigMap, they'll be bzipped and
 base64 encoded.
+
+OS support
+==========
+
+Note that the current testing has been done in RHCOS. We're currently not
+testing in RHEL/CentOS nodes as one can simply run OpenSCAP directly on those
+OSes.
```

Comment:
I know this is an upstream document, but is this going to be our recommendation for OCP, too?

Reply:
I guess so... we haven't really tested in other distros yet. Would be nice! but haven't been able to :confused:

Reply:
I guess what I tried to say is that the statement sounds definitive, like, we consider the current state OK. What do you think about saying "..in the absence of RHEL/CentOS support, one can simply(?) run OpenSCAP..."